plugins/easyimage/plugin.js

```diff
@@ -200,7 +202,11 @@
 				this.replaceWith( '<figure class="' + ( editor.config.easyimage_class || '' ) + '"><img src="' +
 					upload.responseData.response[ 'default' ] + '" srcset="' + srcset + '" sizes="100vw"><figcaption></figcaption></figure>' );
 			}
-		} );
+		};
+
+		addUploadProgressBar( editor, uploadWidgetDefinition );
```

Comment:
Seems like it could reuse `features` feature. What a pity that it must wait for integration of `features` into widget's core 😞

Reply:
This is what I have been thinking of 🙂 That's the reason why I tried to wrap it in a single function, and make it modular so that it could be easily moved.